src/inspector_agent.cc

```diff
@@ -398,10 +383,8 @@ void Agent::OnRemoteDataIO(uv_stream_t* stream,
   inspector_socket_t* socket = static_cast<inspector_socket_t*>(stream->data);
   Agent* agent = static_cast<Agent*>(socket->data);
   if (read > 0) {
-    uv_mutex_lock(&agent->queue_lock_);
-    blink::protocol::String16 str(b->base, read);
-    agent->message_queue_.push_back(str);
-    uv_mutex_unlock(&agent->queue_lock_);
+    String16 str(String16(b->base, read));
```

Comment:
Lets keep code using STL where we can, this would be std::string. Also, no need to execute constructor twice.